includes/apple-exporter/class-settings.php

```diff
@@ -19,6 +19,7 @@ class Settings {
 		'api_secret'      => '',
 		'api_channel'     => '',
 		'api_autosync'    => 'yes',
+		'api_autosync_update'    => 'yes',
 
```

Comment:
This is how the original developers formatted this section. I've chosen not to update it to date.